backend/organization/utility/__init__.py

```diff
@@ -0,0 +1,10 @@
+from enum import Enum 
+
+class RequestStatus(Enum): 
+   PENDING =  1
+   APPROVED = 2
+   REJECTED = 3
```

Comment:
Non-blocker: This is a good approach, but I think the better approach would be using a state machine. So in future if we have different user behavior around status we could easily manage it. https://github.com/viewflow/django-fsm

Of course we can work on this as a separate ticket later. I will leave up to authors of this PR.

Reply:
Neat. I didn't even know about this feature in Django! Do you think the future refactor cost would outweigh the benefit? I don't have a strong POV on whether we should adopt this now or later. I'm inclined to not address, and consider FSM more going forward.

Reply:
On a second thought, I think enums are fine for this PR. We can talk about state machine but not necessary for this work